packages/icon-button/README.md

```diff
@@ -0,0 +1,78 @@
+# React Icon Button
+
+A React version of an [MDC Icon Button](https://github.com/material-components/material-components-web/tree/master/packages/mdc-icon-button).
+
+## Installation
+
+```
+npm install @material/react-icon-button
+```
+
+## Usage
+
+```js
+import React from 'react';
+import IconButton from '@material/react-icon-button';
+import MaterialIcon from '@material/react-material-icon';
+
+class MyApp extends React.Component {
+  render() {
+    return (
+      <IconButton>
+        <MaterialIcon icon='favorite' />
+      </IconButton>
+    );
+  }
+}
+```
+
+You can use any other icon here such as Font Awesome, which is documented more in detail [here](https://github.com/material-components/material-components-web/tree/master/packages/mdc-icon-button#icon-button-toggle-with-font-awesome). If you're using [Google Font's Material Icons](https://material.io/tools/icons/?style=baseline), we recommend using our [Material Icon Component](../material-icon) as shown in the example above.
+
+## Variants
+
+### Icon Button Toggle
+
+If you need to use this component as an Icon Button Toggle, please read [this documentation](https://github.com/material-components/material-components-web/blob/v0.37.1/packages/mdc-icon-button/README.md#icon-button-toggle). The following is an example using data attributes to toggle between on/off states:
+
+```js
+import React from 'react';
+import IconButton from '@material/react-icon-button';
+import MaterialIcon from '@material/react-material-icon';
+
+class MyApp extends React.Component {
+  render() {
+    return (
+      <IconButton
+        className='mdc-icon-button material-icons'
+        aria-label='Add to favorites'
+        aria-pressed='false'
+        data-demo-toggle
```

Comment:
I do not think we should use DOM data attributes here. We should use React props. Is there some weird logic in MDC Web that assumes there are data attributes in the DOM?

Reply:
Ya the icon button foundation is looking for 6 different data attributes. These are copied verbatim:

```
  DATA_TOGGLE_ON_LABEL: 'data-toggle-on-label',
  DATA_TOGGLE_ON_CONTENT: 'data-toggle-on-content',
  DATA_TOGGLE_ON_CLASS: 'data-toggle-on-class',
  DATA_TOGGLE_OFF_LABEL: 'data-toggle-off-label',
  DATA_TOGGLE_OFF_CONTENT: 'data-toggle-off-content',
  DATA_TOGGLE_OFF_CLASS: 'data-toggle-off-class',
```

@williamernest was telling me that there was a suggestion to toggle classes instead of toggle between these data attributes, but those changes haven't been made yet. Do you want to update icon button foundation, and then make appropriate changes here?